pkg/virt-controller/watch/clone/clone.go

```diff
@@ -108,102 +130,104 @@ func (ctrl *VMCloneController) execute(key string) error {
 }
 
 func (ctrl *VMCloneController) sync(vmClone *clonev1alpha1.VirtualMachineClone) (syncInfoType, error) {
-	var syncInfo syncInfoType
+	cloneInfo, err := ctrl.retrieveCloneInfo(vmClone)
+	if err != nil {
+		return syncInfoType{}, err
```

Comment:
In my original comment I meant that the struct is returned without any variables being set. Everything defaults to the type's default. It was like this in the original code but I was wondering if that is a clean solution or has any implications?

Reply:
The returned struct (empty or with defaults) does not play any role here because we return a valued `err`.
If the `sync` func returns an error, the caller will do this:
```
	if err != nil {
		return fmt.Errorf("sync error: %v", err)
	}
```
So no implication here, just compile code :)